src/studio-home/tabs-section/index.jsx

```diff
@@ -23,9 +25,14 @@ const TabsSection = ({
   isPaginationCoursesEnabled,
 }) => {
   const navigate = useNavigate();
+
+  // TODO: this should be a flag in the backend
```

Comment:
This can just be [a frontend config variable](https://github.com/openedx/frontend-app-course-authoring/blob/master/.env). I'm not expecting any backend changes for this PR (no waffle flags). Sorry if that wasn't clear.